src/index.js

```diff
@@ -68,7 +68,7 @@ class TelldusTDToolPlatform {
   }
 }
 
-/*
+/**
```

Comment:
The difference is how it's rendered in editor/IDE, as well as to be consistent with any other comments in the source.

<img width="442" alt="screenshot 2016-10-09 22 54 58" src="https://cloud.githubusercontent.com/assets/121457/19223233/89b2f840-8e73-11e6-994d-4fa62d30ca5e.png">